src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -7,19 +7,27 @@
 
 namespace SprykerSdk\Spryk\Console;
 
+use Symfony\Component\Console\Helper\FormatterHelper;
 use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class SprykDumpConsole extends AbstractSprykConsole
 {
+    protected const COMMAND_NAME = 'spryk:dump';
```

Comment:
is `:list` maybe better?
A dump sounds to me more like a content dump into some file or container.

Reply:
I would agree, but I didn't name it or rename it, so I'm not sure.

.. on the other hand we already use `dump` for other Spryker commands.

Reply:
ok